README.md

```diff
@@ -1,2 +1,45 @@
 # kong-docker
-Kong docker image that easily installs plugins from source code.
+
+The Docker image generated by the Dockerfile on this repository can easily install kong plugins from the source code.
+
+A Kong configuration file containing the plugin names must be included when the container is created.
+
+``` shell
+custom_plugins = <plugin_1>, <plugin_2>
```

Comment:
can we somehow populate this list from the plugins that have been "mounted" into the container?
if need be, cmd could be placed on a segregated file (to abstract the complications of such inference and configuration generation).